.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Welcome! Thanks for your issue. In order for us to investigate your issue, it really helps to make sure we have the information we need. Please fill out the form sections below so we can help you as promptly as we can. Thanks! -->
+
+## Tell us about your runtime:
+
+* **OS**:
```

Comment:
I think we should leave this out to minimize friction. We rarely ever get cases where the OS influences behavior.